utils/validationUserEmail.js

```diff
@@ -1,19 +1,19 @@
-export const validationUserEmail = (inputEmail, type) => {
-  const emailRegex = /[a-z0-9]+@[a-z]+\.[a-z]{2,3}/;
+const EMAIL_REGEX = /[a-z0-9]+@[a-z]+\.[a-z]{2,3}/;
 
+export const validationUserEmail = (inputEmail, type) => {
   let confirmedUserEmail = null;
 
-  inputEmail.length == 0
-    ? alert("이메일을 입력해주세요.")
-    : emailRegex.test(inputEmail)
-    ? (confirmedUserEmail = inputEmail)
-    : alert("올바른 이메일 주소가 아닙니다.");
+  if (inputEmail.length == 0) {
```

Comment:
```suggestion
  if (inputEmail.length === 0) {
```
==는 bad practice중에 하나에요. 
필요한 경우가 있을 수 도 있지만 그런 경우에도 다른 방식으로 처리하는 경우가 많아요.